db/migrate/20200726210802_create_distributors.rb

```diff
@@ -0,0 +1,8 @@
+class CreateDistributors < ActiveRecord::Migration[6.0]
+  def change
+    create_table :distributors do |t|
+      t.string :website_url
```

Comment:
do we want to add name or some other kind of unique identifier?

Reply:
The name lives on the Contact